README.md

```diff
@@ -44,6 +44,10 @@ To run specs:
 make test
 ```
 
+This project uses [Prettier](https://prettier.io/) to format code. When running the lint command above, you may notice errors relating to unexpected code formatting. It's recommended that you install [an editor integration](https://prettier.io/docs/en/editors.html) to automatically format code on save, but you can also resolve these errors automatically from the command-line by running:
+
+npm run lint -- --fix
```

Comment:
can we format this as a code block? (with ```'s)

(I'd do a suggestion but ```'s don't nest)

Reply:
This is what I get for copy-pasting from the preview version of [the original](https://github.com/18F/identity-style-guide#linting) 😄

Reply:
Updated in 4dabc22.